portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,71 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+/*NAVBAR*/
```

Comment:
You can remove this comment. It does not add any value. Since the css classname is "navbar" it is clear what it is.

Reply:
I put it there since the color differentiation makes it easy for me to find the section. Right now it may not be that useful but I know when the file gets longer it will be harder for me to find.